egs2/chime7_task1/asr1/run.sh

```diff
@@ -175,6 +178,10 @@ if [ ${stage} -le 3 ] && [ ${stop_stage} -ge 3 ]; then
     pretrained_affix+="--download_model ${use_pretrained}"
   fi
 
+  if [ $diar_score ]; then
```

Comment:
Maybe this should be changed to `if [ $diar_score -eq 1 ]; then`, otherwise this condition will always evaluate to True.